src/components/accessible-svg/AccessibleSVG.stories.tsx

```diff
@@ -13,6 +13,6 @@ export const IconExample = (props: SVGProps) => (
 );
 
 export default {
-    title: 'Components|AccessibleSVG',
+    title: 'Components/AccessibleSVG',
```

Comment:
Had to change this because it treats it as the title of the component instead of creating a hierarchy

Reply:
<img width="397" alt="image" src="https://github.com/box/box-ui-elements/assets/103291617/8e5cea25-e5f4-464a-8c7d-dc35a4607f2b">
I undid the the `AccessibleSVG` component to show you what it looks like without the change in comparison to the change.

With the change, it should closer to how we already have it. Without the change it doesnt group the components together and makes them their own section if there was more than one `Components|Something` there